tests/geomultiscale-mapping/RadialGeoMultiscale.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration dimensions="3" experimental="on">
```

Comment:
So, we proceed with the global dimensions for now, and we move the dimensions to the participants later (that's on my list at the moment).